contracts/GuardiansRegistration.sol

```diff
@@ -125,8 +125,8 @@ contract GuardiansRegistration is IGuardiansRegistration, ManagedContract {
 	 * Governance
 	 */
 
-	function migrateGuardians(address[] calldata guardiansToMigrate, IGuardiansRegistration previousContract) external override onlyInitializationAdmin {
-		require(previousContract != IGuardiansRegistration(0), "previousContract must not be the zero address");
+	function migrateGuardians(address[] calldata guardiansToMigrate, IGuardiansRegistrationPreviousVersion previousContract) external override onlyInitializationAdmin {
```

Comment:
Since the code is for migration from a specific version, should it be hardcoded?

Reply:
We can deploy a new contract with the same interface if we want additional migrations, but it is unlikely